featurebyte/service/feature_list.py

```diff
@@ -320,6 +351,7 @@ async def create_document(self, data: FeatureListServiceCreate) -> FeatureListMo
                 "features_entity_lookup_info": entity_relationship_data.features_entity_lookup_info,
             }
         )
+        await self._move_feature_cluster_to_storage(document)
```

Comment:
Can the remaining steps below fail? If so might be better to add a step to clean up the remote file when that occurs.